conf/airframes/tudelft/disco_modified.xml

```diff
@@ -0,0 +1,635 @@
+        <define name="AHRS_PROPAGATE_LOW_PASS_RATES" value="FALSE"/> <!-- apply a low pass filter on rotational velocity"-->
+        <define name="AHRS_BIAS_UPDATE_HEADING_THRESHOLD" value="0.174533" unit="rad"/> <!-- don't update gyro bias if heading deviation is above this rotation threshold"-->
+        <define name="AHRS_HEADING_UPDATE_GPS_MIN_SPEED" value="3.0" unit="m/s"/> <!-- CAREFULL,  Don't update heading from GPS course if GPS ground speed is below is this threshold in m/s" -->
+        <!-- Some insights https://lists.nongnu.org/archive/html/paparazzi-devel/2013-10/msg00126.html -->
+        <define name="AHRS_GRAVITY_HEURISTIC_FACTOR" value="0.5"/> <!-- TODO: determine Default is 30. Reduce accelerometer cut-off frequency when the vehicle is accelerating: norm(ax,ay,az) 9,81 m/s2. WARNING: when the IMU is not well damped, the norm of accelerometers never equals to 9,81 m/s2. As a result, the GRAVITY_HEURISTIC_FACTOR will reduce the accelerometer bandwith even if the vehicle is not accelerating. -->
+        <define name="AHRS_ICQ_IMU_ID" value="ABI_BROADCAST"/> <!-- ABI sender id of IMU to use -->
+        <define name="AHRS_ICQ_MAG_ID" value="MAG_CALIB_UKF_ID" /><!-- for when using the mag_clib_ukf -->
+    </module>
+
+    <module name="ins" type="alt_float"/>
+
+    <module name="control" type="new"/>
+    <module name="navigation"/>
+    <!--<module name="imu_quality_assessment"/>--><!--needs fixing for FW fix test then disable after initial tuning-->
+    <module name="send_imu_mag_current"/> <!-- Can be Disabled after one time callibration, quite the same for all Discos -->
+
+    <!-- Also used if QNH needed -->
+    <module name="air_data">
+     <define name="CALC_AIRSPEED" value="FALSE"/><!--  we set use airspeed elsewhere-->
+     <define name="CALC_TAS_FACTOR" value="FALSE"/>
+     <define name="CALC_AMSL_BARO" value="TRUE"/>
+    </module>
+
+    <module name="geo_mag"/>
+    <module name="nav" type="line"/>
+    <module name="nav" type="line_border"/>
+    <module name="nav" type="line_osam"/>
+    <module name="nav" type="survey_polygon">
+      <define name="POLYSURVEY_DEFAULT_DISTANCE" value="40" unit="m"/> <!-- Make it your fit for plan and camera type -->
+    </module>
+    <module name="nav" type="survey_poly_osam"/>
+    <module name="nav" type="smooth"/>
+    <module name="nav" type="vertical_raster"/>
+    <module name="nav" type="flower"/>
+
+  </firmware>
+
+ 
+
+<!-- ************************* ACTUATORS ************************* -->
+  <servos>
+    <servo name="S_THROTTLE_LEFT" no="0" min="1100" neutral="1110" max="1900"/>
+    <servo name="S_THROTTLE_RIGHT" no="1" min="1100" neutral="1110" max="1900"/>
+    <servo name="S_ELEVON_LEFT" no="2" min="1100" neutral="1500" max="1900"/>
+    <servo name="S_ELEVON_RIGHT" no="3" min="1900" neutral="1500" max="1100"/>
+  </servos>
+
+  <section name="ServoPositions">
+    <define name="ThrottleHigh()" value="(ap_state->commands[COMMAND_THROTTLE]>9600/2)" />
```

Comment:
you should use `imcu_get_command(COMMAND_THROTTLE)` from `inter_mcu.h` instead of accessing directly the `ap_state` structure. It doesn't make a big difference in your case without RTOS, but for people using boards with RTOS, it's not thread safe.

Reply:
I've made the change to imcu_get_command, is calling the function as I did sufficient?

Reply:
`imcu_get_command(COMMAND_THROTTLE)` is only equivalent to `ap_state->commands[COMMAND_THROTTLE]`, so you should keep the `> (9600/2)` after this

Reply:
I re-added the last bit, thanks!